### PR TITLE
Migrate GCR to GAR

### PR DESCRIPTION
**Pull request by matthew-robinson-ons**
# What and why?
Migrates the image from google container registry to google artifacts registry.
# How to test?
Deploy your environment
Deploy this pr with the config branch https://github.com/ONSdigital/ras-rm-config/pull/274
Update the deployment yaml to point to the correct image location `europe-west2-docker.pkg.dev/ons-ci-rmrasbs/images/`
Check image is in the correct project artifact registry

N.B. there are references to both survey and pubsub emulator within collex. We will need to loop back and update these once those have been moved to GAR. 

# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-1349

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -51,9 +52,9 @@ jobs:
           credentials_json: ${{ secrets.GCR_KEY }}
       - name: Setup Google Cloud SDK
         uses: google-github-actions/setup-gcloud@v2
-      - name: Configure GCR
-        run: |
+      - run: |
           gcloud auth configure-docker
```
Comment (matthew-robinson-ons) [inline .github/workflows/main.yml:56]: Pubsub elulator and survey are still within the GCR. Auth is needed for both projects

File: .github/workflows/manual.yml
Comment (matthew-robinson-ons) [inline .github/workflows/manual.yml:1]: Not sure if this file is still relevant? Have updated to line up with our other workflow files

Comment (pricem14pc) [reply .github/workflows/manual.yml:1]: Yes, this is needed.

**Review by pricem14pc — APPROVED** [review]
PR images handled as expected and eyeballed the main flow.

**Review by arroyoAle — APPROVED** [review]
Deploys correctly from GAR when the yaml is updated and image works

**Pull request merged**